ansible/roles/bigbluebutton/molecule/default/molecule.yml

```diff
@@ -12,6 +12,7 @@ lint: |
 platforms:
   - name: ubuntu1604-instance
     box: generic/ubuntu1604
```

Comment:
Not important for now, but just a note that: (emphasis mine; taken from [here](https://ubuntu.com/16-04))

> Ubuntu 16.04 LTS is supported until 2024 through Canonical’s Extended Security Maintenance (ESM) product. Xenial enters the ESM period in **April 2021**, with security patches provided for an additional three years beyond the traditional five-year standard support.

Related BBB issue: https://github.com/bigbluebutton/bigbluebutton/issues/8474#issuecomment-802798671


Created an issue for it: https://github.com/PrivacyLx/privacylx-issue-tracker/issues/97

Reply:
Thanks for opening an issue.

Reply:
Thanks!